src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute-updates/jj-dispute-updates.component.ts

```diff
@@ -44,8 +44,7 @@ export class JjDisputeUpdatesComponent {
     "PleaCode": "Plea Code",
     "CountNo": "Count No",
     "RequestTimeToPay": "Request Time To Pay",
-    "RequestReduction": "Request Reduction",
-    "RequestCourtAppearance": "Request Court Appearance"
```

Comment:
So, do we not want to include the RequestCourtAppearance update made by the disputant anymore?

Reply:
RequestCourtAppearance  is not part of count

Reply:
It is actually part of the dispute count.

Reply:
![image](https://github.com/user-attachments/assets/3f273601-448f-429f-83d4-e79de5c65154)

Reply:
From the screenshot provided, you can see it is not part of count. We need to check the highlighted part rather tan under the count
 "FineReductionReason": "TCVP-2934 Test - Fine Reduction",
    "TimeToPayReason": "TCVP-2934 Test - Time To Pay",
    "**RequestCourtAppearance**": "N",  -- this is the correct value which needs to be checked
    "SignatoryName": "Givenone Surname",
    "SignatoryType": "D",
    "DisputeCounts": [{
            "PleaCode": "G",
            "CountNo": 1,
            "RequestTimeToPay": "Y",
            "RequestReduction": "Y",
            "RequestCourtAppearance": "UNKNOWN"
        }, {
            "PleaCode": "G",
            "CountNo": 2,
            "RequestTimeToPay": "N",
            "RequestReduction": "N",
            "RequestCourtAppearance": "UNKNOWN"
        }, {
            "PleaCode": "G",
            "CountNo": 3,
            "RequestTimeToPay": "N",
            "RequestReduction": "N",
            "RequestCourtAppearance": "UNKNOWN"
        }
    ],

Reply:
Ah I see. There is a separate RequestCourtAppearance flag on the dispute model level which is the one we're actually using despite we have one for each on count model as well. If we are using the flag on the dispute model only, we should probably remove the other one from the count at some point because it's confusing.